CLIENT-SPECIFICATION.md

```diff
@@ -205,20 +202,21 @@ The [`LC_MESSAGES` environment variable](https://www.gnu.org/software/gettext/ma
 
 Here's an example of how the lookup should be done on `linux` having set `LANG=it` and `LANGUAGE="it:fr:en"`:
 
-    1. pages.it/linux/some-page.md  -> does not exist
-    2. pages.fr/linux/some-page.md  -> does not exist
-    3. pages/linux/some-page.md     -> does not exist
-    4. pages.it/common/some-page.md -> does not exist
-    5. pages.fr/common/some-page.md -> does not exist
-    6. pages/common/some-page.md    -> FOUND!
+```txt
```

Comment:
Not sure why this is in a code fence? If it really has to be, try `markdown` instead of `txt`

Reply:
Not sure why I went with `txt` initially for the codeblock, maybe to avoid markdown lint errors I guess 🤷🏽‍♂️ .

```suggestion
```md
```

Reply:
Is this to align the `→`, @kbdharun? In which case it might be worth reformatting as a table, since a code fence here looks odd.